crm_task/readme/USAGE.rst

```diff
@@ -0,0 +1,5 @@
+To use this module, you need to:
+
+1. Go to Lead
+2. A new button allows you to create/access project tasks
+3. A new page list all related tasks from the opportunity
```

Comment:
This feature doesn't exist. You could explain how to link a task with a lead/opportunity.